pipelines/DRP.yaml

```diff
@@ -1,16 +1,158 @@
-description: DRP specialized for DECam
+description: The DRP pipeline specialized for the DECam instrument
 instrument: lsst.obs.decam.DarkEnergyCamera
 imports:
-  - location: $PIPE_TASKS_DIR/pipelines/DRP.yaml
-    exclude:
-      - sourceTable
-      - objectTable
-      - isr
-  - location: $AP_PIPE/pipelines/RunIsrWithCrosstalk.yaml
-# This DRP pipeline uses an obs_decam specific ISR configuration that
-# requires crosstalk sources to be pre-generated via the
-# RunIsrForCrosstalkSources.yaml pipeline.  This applies overscan
-# correction and writes a new dataset that is then used as needed for
-# the inter-chip crosstalk solutions.  Unless a specific overscan
-# change is required, it may be best to generate these crosstalk
-# sources soon after ingest to allow them to be readily available.
+  location: $PIPE_TASKS_DIR/pipelines/DRP.yaml
+tasks:
+  isrForCrosstalkSources:
+    class: lsst.ip.isr.IsrTask
+    config:
+    - connections.outputExposure: overscanRaw
+      doOverscan: true
+      doAssembleCcd: false
+      doBias: false
+      doVariance: false
+      doLinearize: false
+      doDefect: false
+      doNanMasking: false
+      doDark: false
+      doFlat: false
+      doFringe: false
+      doInterpolate: false
+  isr:
+    class: lsst.ip.isr.IsrTask
+    config:
+    - connections.crosstalkSources: overscanRaw
```

Comment:
I am again confused about how some of the sub-thingies have hyphens and others do not... here and in calibrate, too

Reply:
The YAML format is surprisingly versatile - this surprised me too! So, either only the first indented line can have a hyphen, or all of them can. I don't have a preference, however, when using `pipetask build` to display the pipeline, it formats it like this. For that reason, I opted to format this pipeline to match the output given by `pipetask build`.

Reply:
Interesting. We really do need a yaml style guide, don't we... you have 2-space indents, at least, so I'll let you do what you like with regards to hyphens and whatnot.

Reply:
For consistency (e.g., with the obs_subaru YAML pipeline file), I've removed all opening config hyphens here.